packages/myst-common/src/index.ts

```diff
@@ -24,6 +24,7 @@ export {
   ParseTypesEnum,
   TargetKind,
 } from './types.js';
+export type { IExpressionResult, IExpressionError, IExpressionOutput } from './types.js';
```

Comment:
@agoose77 can you add changesets for these as you are going! :)

Reply:
Yep! It's on my TODO, ha!